stores/message.js

```diff
@@ -189,7 +201,11 @@ export const useMessageStore = defineStore({
           }
         }
 
-        this.byUserList[userid] = messages
+        if (Array.isArray(this.byUserList[userid]) && active) {
```

Comment:
I don't understand why we have the `&& active` check here and below.  Can you explain?

Reply:
If active is true, we might not get all the posts of the user, so in order not to lose old posts, we will use that response and update items we already have in store

Reply:
If a post has been deleted on the server, we **want** to remove it on the client.

Now that you have moved the active count check into a separate function which doesn't affect the state in the store, can we just revert to setting `this.byUserList[userid] = messages`?